docs/installation/installing-tentacles/polling-tentacles-web-sockets.md

```diff
@@ -0,0 +1,91 @@
+
+If you do not yet know the thumbprint of your SSL certificate, you can find it by using your browser's developer tools.
+
+If you get your other application or an error, check the [server log file](/reference/log-files.md) to ensure the server is listening on the
+expected prefix (look for `Listener started`)
+
+## Tentacle Setup
+The setup of a WebSocket tentacle is the same as a TCP polling tentacle, except for the thumbprint and the command line option to specify the communications address.
+
+### Registering
+When issuing the `register-with` command during tentacle setup, omit the `--server-comms-port` parameter and specify the `--server-web-socket <address>` parameter. The address to use is the listen prefix (replacing `+` with the hostname) and `https` replaced with `wss` (e.g. `wss://MyServer:443/OctopusComms`). For example:
+```powershell
+.\Tentacle.exe register-with --instance MyInstance --server "https://MyServer/"  --server-web-socket "wss://MyServer:443/OctopusComms" --comms-style TentacleActive --apikey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --environment "Test" --role "Web" 
+```
+
+### Changing an Existing Tentacle
+To change an existing tentacle to poll using WebSockets, run the following commands:
+```powershell
+.\Tentacle.exe service --instance MyInstance --stop
+.\Tentacle.exe configure --reset-trust
+.\Tentacle.exe register-with --instance MyInstance --server "https://MyServer/" --server-web-socket "wss://MyServer:443/OctopusComms" --comms-style TentacleActive --apikey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO" --environment "Test" --role "Web" 
+.\Tentacle.exe service --instance MyInstance --start
+```
+
+### High Availablity
+When issuing the `poll-server` command to add additional nodes to poll, `--server-comms-port` parameter and specify the `--server-web-socket <address>` parameter. For example:
+
+```powershell
+poll-server --instance MyInstance --server "https://MyServer/"  --server-web-socket "wss://MyServer:443/OctopusComms" --apikey "API-CS0SW5SQJNLUBQCUBPK8LZY3KYO"
+```
+## More Information
+
+### SSL Certificate
+Windows will need to be configured with an SSL certificate on the select address and port. Usually this is done by the other application sharing the port.
+The certificate does _not_ need to be trusted by the server or tentacle machine. The certificate also does not need to match the hostname.
+Self signed certificates are accepted, no chain of trust is required. It does need to be installed into the Personal certificate store of the Machine account.
+
+If you need to generate a self signed certificate, this can be done by issuing the following PowerShell command. Take note of the thumbprint generated.
+```powershell
+New-SelfSignedCertificate -Subject "CN=Example Website" -CertStoreLocation "Cert:\localMachine\My" -KeyExportPolicy Exportable
+```
+
+If your chosen certificate has not yet been associated with the selected address and port, use the `netsh` tool to install it. For example:
```

Comment:
Maybe mention that if the portal is using this SSL cert, this will have been done